config/locales/en.yml

```diff
@@ -838,7 +838,7 @@ en:
       #EmsCloud:                 Cloud Provider
       EmsCluster:               Cluster / Deployment Role
       EmsClusterPerformance:    Performance - Cluster
-      EmsContainer:             Container Provider
+      EmsContainer:             Containers Provider
```

Comment:
There already was an entry under 'model' but I added the 's' to keep consistent with what usually was the display name for EmsContainer